src/ad-bidders/prebid/index.ts

```diff
@@ -413,6 +414,25 @@ export class PrebidProvider extends BidderProvider {
 		return {};
 	}
 
+	private configureDSA(): object {
+		if (context.get('options.dsa.enabled')) {
+			return {
+				ortb2: {
+					regs: {
+						ext: {
+							dsa: {
+								required: 1,
```

Comment:
```suggestion
								dsarequired: 1,
```
DSA field names changed in the meantime 🙈